detection_rules/devtools.py

```diff
@@ -354,3 +356,217 @@ def rule_survey(ctx: click.Context, query, date_range, dump_file, hide_zero_coun
         json.dump(details, f, indent=2, sort_keys=True)
 
     return survey_results
+
+
+@dev_group.group('gh-release')
+def gh_release_group():
+    """Commands to manage GitHub releases."""
+
+
+@gh_release_group.command('create-ml')
+@click.argument('directory', type=click.Path(dir_okay=True, file_okay=False))
+@click.option('--gh-token', '-t', default=getdefault('gh_token'))
+@click.option('--repo', '-r', default='elastic/detection-rules', help='GitHub owner/repo')
+@click.option('--release-name', '-n', required=True, help='Name of release')
+@click.option('--description', '-d', help='Description of release to append to default message')
+@click.pass_context
+def create_ml_release(ctx, directory, gh_token, repo, release_name, description):
+    """Create a GitHub release."""
+    import re
+
+    # ML-DGA-20201129-25
+    pattern = r'^(ML-DGA|ML-experimental-detections)-20\d\d[0-1]\d[0-3]\d-\d+$'
```

Comment:
i know this is less precise, but it's simpler to read and debug and will still get the job done
```suggestion
    pattern = r'^(ML-DGA|ML-experimental-detections)-\d{4}\d{2}\d{2}-\d+$'
```